README.md

```diff
@@ -1,2 +1,39 @@
 # Portfolio-Ch-Ahtesham-Ul-Haq
 Setup and Mobile version skeleton
+
+Hi, I simple coded the mobile version with the help of flexbox and figma features.
+It is a structure where we decorate and furnishe further. 
+
+Built with languages
+HTML & CSS
+Frameworks
+Technologies used
+Linters, HTML, and CSS
+
+To get a local copy up and running follow these simple example steps.
+
+Prerequisites
+Setup
+Install
+Usage
+Run tests
+Deployment
+
+👤 Author Ahtesham Ul Haq
+
+GitHub:https://www.linkedin.com/in/ch-ahtesham-ul-haq-b5318b152/
+
+🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the issues page.
+
+Show your support
+Give a ⭐️ if you like this project!
+
+Acknowledgments
+Hat tip to anyone whose code was used
+Inspiration
+etc
+📝 License
+This project is MIT licensed.
```

Comment:
[OPTIONAL]

For some reason, the README is looking strange, check the picture below:

![image](https://user-images.githubusercontent.com/20567503/177659400-b0b27e8c-c587-4277-bcd8-f7c69dfeef6d.png)

I recommend that you go back to the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by Microverse and edit yours again using it.